.gitignore

```diff
@@ -11,6 +11,9 @@ data/
 # Dotenv file
 .env
 
+# Firebase secrets
+firebase.json
```

Comment:
We should add a template for firebase.json

Reply:
Firebase.json is a document generated by Firebase. I don't this it would really make sense to generate a template for this file.